distributed/utils.py

```diff
@@ -31,10 +30,9 @@ def funcname(func):
         return str(func)
 
 
-def get_ip():
-    return [(s.connect(('8.8.8.8', 80)), s.getsockname()[0], s.close())
-        for s in [socket.socket(socket.AF_INET, socket.SOCK_DGRAM)]][0][1]
-
+def get_ip(host='8.8.8.8', port=80):
+    return [(s.connect((host, port)), s.getsockname()[0], s.close())
+            for s in [socket.socket(socket.AF_INET, socket.SOCK_DGRAM)]][0][1]
```

Comment:
BTW I think we might want to fallback to `socket.gethostbyname(socket.gethostname())` if `host` is left to the default `8.8.8.8` and is unreachable (not connected to internet). But this should probably be addressed in a separate PR.